### PR TITLE
Allow detection of person name written in latin script for hindi bots

### PR DESCRIPTION
**Pull request by ashutoshsingh0223**
ML-835
detection of person name written in latin script for hindi bots

### CODE REVIEW

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

File: ner_v1/detectors/textual/name/name_detection.py [hunk 1]
```diff
@@ -344,23 +353,25 @@ def get_hindi_names_without_regex(self, text):
...
             text = u'प्रतिक श्रीदत्त जयराओ'
             get_hindi_names_without_regex(text=text)
-
             >> [{first_name: u"प्रतिक", middle_name: u"श्रीदत्त", last_name: u"जयराओ"}], [ u'प्रतिक श्रीदत्त जयराओ']
-
         """
-
         text = self.replace_stopwords_hindi(text)
+        word_list = [word for word in text.split(" ") if word not in COMMON_HINDI_WORDS_OCCURING_WITH_NAME]
+        if word_list:
```
Comment (amansrivastava17) [inline ner_v1/detectors/textual/name/name_detection.py]: text = " ".join([word for word in text.split(" ") if word not in COMMON_HINDI_WORDS_OCCURING_WITH_NAME])
if not text:
    return [], []

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**